layouts/terms/single.html

```diff
@@ -1,37 +1,78 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>{{ .Site.Title }}</title>
-    <link rel="canonical" href="{{ .Permalink }}" />
-  </head>
-  <body>
 
-    {{ partial "header.html" . }}
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <link rel="stylesheet" href="/main.css" type="text/css">
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+    integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
+    integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
+    crossorigin="anonymous"></script>
+  <script src="/toggle.js"></script>
 
-    <main>
-        
+  <!-- Primary Meta Tags -->
+  <title>{{ .Site.Title }}</title>
+  <meta name="title" content="{{ .Site.Title }}">
+  <meta name="description"
+    content="{{if .Site.Params.description}}{{ .Site.Params.description }}{{else}}{{.Description}}{{end}}">
```

Comment:
```suggestion
  <meta name="description"
    content="{{ substr .Summary 0 150 }}…">
```

@dr5hn Try this ☝️ - the `.Summary` variable is some hugo magic that takes the `.Content` and strips out formatting. So if we substring it to 150 chars and add an ellipsis I think we get a pretty good description without adding an extra required param to the spec.

What do you think?

Reply:
@ruf-io Great, Thanks for reference.

But i did it better way, I used
`{{ .Summary | truncate 150  }}`

truncate - Does truncating chars upto a limit and adds an ellipsis at the end by it self.